bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/enums/HeatpumpChannel.java

```diff
@@ -938,7 +938,7 @@ public enum HeatpumpChannel {
      * Control signal circulating pump
      * (original: Steuersignal Umwälzpumpe)
      */
-    CHANNEL_HZIO_PWM(183, "controlSignalCirculatingPump", NumberItem.class, Units.PERCENT, false, null),
+    CHANNEL_HZIO_PWM(183, "controlSignalCirculatingPump", NumberItem.class, null, false, null),
```

Comment:
We need to keep the percent unit here. Based on the [LoxWiki](https://loxwiki.atlassian.net/wiki/spaces/LOX/pages/1533935933/Java+Webinterface) this channel provides a percent value, and the provided value needs to be divided by 10. This happens automatically for percent values, but won't happen without the unit. See https://github.com/openhab/openhab-addons/blob/dc3ca47b1218747f2b3770a3d7a899f5f91b8e05/bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/ChannelUpdaterJob.java#L135-L152

Reply:
Changed back to Units.PERCENT